packages/api/src/app-router.ts

```diff
@@ -35,6 +41,26 @@ const GEOLOCATION_ENDPOINTS = [
   "geolocate",
 ];
 
+const CORS_WHITELIST = [
+  "https://livepeer.com",
+  "https://livepeer.monster",
+  "https://explorer.livepeer.org",
+  "http://localhost:3000",
```

Comment:
Actually the localhost here is unnecessary afaik since browsers already allow cors from localhost by default. I'll remove the 3000 as well.

Reply:
Actually, maybe it only worked on localhost becaus we had this here? But probably browsers/the cors library just ignored the port cause I doubt all hackers that ever depended on it ran their applications on port 3000.

Nevertheless, I'll try to improve this list as well. I don't think we should be running production with CORS enabled for localhost. Probably enable it only development and somehow for the box?

Reply:
Created the `LP_API_CORS_JWT_ALLOWLIST` which should be set in custom environments where the dashboard is going to be somewhere different from `locahost:3000`. Already set it in the staging environment like this: https://github.com/livepeer/livepeer-infra/blob/f43702e3fdf34adff59090629a76a3411b478e62/values/staging/livepeer.values.yaml#L12

WDYT? Think we can set an env like that in the box with the custom `3004` port?

Reply:
> Created the `LP_API_CORS_JWT_ALLOWLIST` which should be set in custom environments where the dashboard is going to be somewhere different from `locahost:3000`. Already set it in the staging environment like this: [livepeer/livepeer-infra@`f43702e`/values/staging/livepeer.values.yaml#L12](https://github.com/livepeer/livepeer-infra/blob/f43702e3fdf34adff59090629a76a3411b478e62/values/staging/livepeer.values.yaml#L12)

I was concocting the same theory while flying back today ^_^ :+1: a set of csv domains to be populated through environment variables. It accepting regex patterns is super cool :star2: :star_struck: 

> 
> WDYT? Think we can set an env like that in the box with the custom `3004` port?

it should theoretically be possible, but since MistController is the one overtaking spawns, might need to confirm! Maybe @iameli might know right off about this behaviour